crates/forge/tests/e2e/running.rs

```diff
@@ -385,3 +385,42 @@ fn using_corelib_names() {
         Tests: 4 passed, 0 failed, 0 skipped
         "#});
 }
+
+#[test]
+fn should_panic() {
+    let temp = assert_fs::TempDir::new().unwrap();
+    temp.copy_from("tests/data/should_panic_test", &["**/*.cairo", "**/*.toml"])
+        .unwrap();
+
+    let snapbox = runner();
+
+    snapbox
+        .current_dir(&temp)
+        .assert()
+        .success()
+        .stdout_matches(indoc! { r#"
+        Collected 6 test(s) and 2 test file(s)
+        Running 0 test(s) from src/lib.cairo
+        Running 6 test(s) from tests/should_panic_test.cairo
+        [PASS] should_panic_test::should_panic_test::should_panic_no_data
+
+        Success data:
+            original value: [0], converted to a string: []
+
+        [PASS] should_panic_test::should_panic_test::should_panic_check_data
+        [PASS] should_panic_test::should_panic_test::should_panic_multiple_messages
+        [FAIL] should_panic_test::should_panic_test::should_panic_with_non_matching_data
+
+        Failure data:Incorrect panic data
+            Actual:    [8111420071579136082810415440747] (failing check)
+            Expected:    [0] ()
```

Comment:
Can we format it so it looks like this instead?
```suggestion
        Failure data:
            Incorrect panic data!
            Actual:    [8111420071579136082810415440747] (failing check)
            Expected:    [0] ()
```